packages/dscc-validation/src/schemas.ts

```diff
@@ -367,17 +367,12 @@ export const configSchema = {
           enum: ["FONT_COLOR"]
         },
         defaultValue: {
-          anyOf: [
-            { type: "string" },
-            {
-              type: "object",
-              additionalProperties: false,
-              required: ["value"],
-              properties: {
-                value: { type: "string" }
-              }
-            }
-          ]
+          type: "object",
+          additionalProperties: false,
+          required: ["color"],
+          properties: {
+            color: { type: "string" }
```

Comment:
also add opacity?
Same for the rest of the changes

Reply:
done